### ISSUE TITLE
dhtSync fails with TypeError

### ISSUE THREAD

**Issue by matthme** [holochain/tryorama#211]
```
error occurred during test run: TypeError: The first argument must be of type string or an instance of Buffer, ArrayBuffer, or Array or an Array-like Object. Received undefined
    at Function.from (node:buffer:319:9)
    at maxargs (file:///private/tmp/forum/node_modules/js-base64/base64.mjs:64:23)
    at Object.fromUint8Array (file:///private/tmp/forum/node_modules/js-base64/base64.mjs:79:71)
    at encodeHashToBase64 (file:///private/tmp/forum/node_modules/@holochain/client/lib/utils/base64.js:22:23)
    at file:///private/tmp/forum/node_modules/@holochain/tryorama/lib/util.js:58:17
    at /private/tmp/forum/node_modules/lodash/_baseOrderBy.js:39:14
    at arrayMap (/private/tmp/forum/node_modules/lodash/_arrayMap.js:16:21)
    at /private/tmp/forum/node_modules/lodash/_baseOrderBy.js:38:20
    at /private/tmp/forum/node_modules/lodash/_baseMap.js:17:23
    at /private/tmp/forum/node_modules/lodash/_createBaseEach.js:24:11 {
  code: 'ERR_INVALID_ARG_TYPE'
}
```

tryorama version: `0.17.0-dev.0`
holochain version: `0.4.0-dev.4`

Example failure case: https://github.com/lightningrodlabs/tools-library/blob/ed4a4e695be779d6bddf3f5b26c0419103a804b1/tests/src/tools/library/tool.test.ts#L142

Reproduction:
1. Clone `https://github.com/lightningrodlabs/tools-library`
2. `nix develop`
3. `npm install`
4. `npm run build:happ`
5. `npm run test`

**Comment by c12i** [comment]
Issue still occurs on `holochain 0.4.0-dev.7`

**Comment by jost-s** [comment]
Confirmed, happening on CI https://github.com/holochain/tryorama/actions/runs/9491077453/job/26155803415?pr=213#step:13:116

**Comment by jost-s** [comment]
fixed by #213
